install_files/ansible-base/build-deb-pkgs.yml

```diff
@@ -21,5 +21,6 @@
   roles:
     - build-securedrop-app-code-deb-pkg
     - { role: build-ossec-deb-pkg, tags: [ "ossec" ] }
+    - build-grsec-metapackage-deb-pkg
```

Comment:
Even though the grsec meta package doesn't take long to build we could add a role for it so it could be skipped. Thoughts?